.travis.yml

```diff
@@ -13,6 +16,8 @@ addons:
     - libpng-dev
     - libfreetype6-dev
     - libcairo2-dev
+    - libglu1-mesa-dev
```

Comment:
@jwiggins I'm guessing here. Do you have better information on what extra packages should be installed to get OpenGL libraries?

Reply:
I do not